base_layer/wallet/tests/output_manager_service/service.rs

```diff
@@ -885,10 +885,15 @@ fn test_utxo_validation() {
         loop {
             futures::select! {
                 event = event_stream.select_next_some() => {
-                    match event.unwrap() {
-                        OutputManagerEvent::TxoValidationTimedOut(_) => {
-                            timeouts+=1;
-                         },
+                    match event {
```

Comment:
So this doesn't need to change or be reverted but just a philosophical comment for the future. The removal of the `unwrap()` in the `tari_base_node::transaction_monitor` definitely needed to happen but in tests it is, in my opinion, preferable to leave an unwrap rather than gracefully deal with error like you have done here so that the test panics if there is a problem. In a test we prefer to see the panics explicitly.